app/components/search_bar/search_bar.ios.js

```diff
@@ -116,9 +116,9 @@ export default class SearchBarIos extends Component {
                     {...this.props}
                     ref='search'
                     placeholderCollapsedMargin={this.state.placeholderWidth}
-                    placeholderExpandedMargin={20}
-                    searchIconCollapsedMargin={this.state.placeholderWidth + 10}
-                    searchIconExpandedMargin={10}
+                    placeholderExpandedMargin={30}
```

Comment:
looks like this

![image](https://user-images.githubusercontent.com/6757047/27136510-801b93d0-50e9-11e7-98ef-fdde2cc81862.png)